src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/GestureResult.cs

```diff
@@ -13,7 +13,7 @@ public static GestureResult Get(QueryEx textBlock)
 
 		public static GestureResult Parse(string text)
 		{
-			var regex = new Regex(@"(?<elt>[\w_]+)@(?<x>[\d\.]+),(?<y>[\d\.]+)");
+			var regex = new Regex(@"(?<elt>[\w_]+)(-(?<state>\w+))?@(?<x>[\d\.]+),(?<y>[\d\.]+)");
```

Comment:
Regex are expensives. Maybe you should parse the string _more manually_.

Reply:
It's for UI tests